src/providers/solr-search.php

```diff
@@ -70,27 +69,25 @@ function parse_query_response( $responseString ) {
     $docs = $parsedResult->response->docs;
     foreach ( $docs as $doc ) {
       $item = new SearchResultItem();
-      $item->id = $doc->id;
+      $item->id = $doc->id[0];
```

Comment:
@rraub Well, my `schema.xml` didn't explicitly allow arrays in these fields, but the JSON returned from the SOLR server looks something like this:
```
{
    "title":["art%3A10.1007%2Fs10460-008-9140-2.txt"],
    "body":["﻿..."],
    "id":["5a44f242-eae3-4282-99fb-9e77b4f2b209"]}]
}
```
So I "normalized" these fields.